### DISCUSSION TITLE
handbook/test-runners/playwright-test/

### DISCUSSION THREAD

**Discussion by giscus** [serenity-js/serenity-js#1891] [Comments]
# handbook/test-runners/playwright-test/

Playwright Test is a test runner designed specifically to accommodate the needs

https://serenity-js.org/handbook/test-runners/playwright-test/

**Comment by jbpros** [comment]
Hey -- I believe there is a small error In the "Defining Playwright Test scenarios" section: the code example should display `import { describe, it } from '@serenity-js/playwright-test'` instead of `import { describe, it } from '@playwright/test'`

**Reply by jbpros** [reply]
Ignore me, that's explained right in the next section. I was confused as I was typing along in my editor and the compiler didn't like that erroneous import from playwright.

**Reply by jan-molak** [reply]
No worries, @jbpros! If you think we can make the instructions more clear please let me know
